src/components/PersonLink/PersonLink.tsx

```diff
@@ -0,0 +1,20 @@
+import { Link } from 'react-router-dom';
+import classNames from 'classnames';
+import { Person } from '../../types';
+
+type Props = {
+  person: Person;
+};
+
+export const PersonNavigate: React.FC<Props> = ({ person }) => {
```

Comment:
```suggestion
export const PersonLink: React.FC<Props> = ({ person }) => {
```
The name should respect the file name